libcst/tests/test_batched_visitor.py

```diff
@@ -16,57 +16,57 @@ def test_simple(self) -> None:
         mock = Mock()
 
         class ABatchable(BatchableCSTVisitor):
-            def visit_Pass(self, node: cst.Pass) -> None:
-                mock.visited_a()
-                object.__setattr__(node, "a_attr", True)
+            def visit_Del(self, node: cst.Del) -> None:
+                object.__setattr__(node, "target", mock.visited_a())
```

Comment:
I had to change to using `Del`, because `Pass` didn't have two fields available to reproduce the original test.

due to the slots, arbitrary fields can no longer be set. Existing fields need to be (ab)used.